nicegui/storage.py

```diff
@@ -149,10 +151,26 @@ def general(self) -> Dict:
         """General storage shared between all users that is persisted on the server (where NiceGUI is executed)."""
         return self._general
 
+    @property
+    def client(self) -> ObservableDict:
+        """Client storage that is persisted on the server (where NiceGUI is executed) on a per client
+        connection basis.
+
+        The data is lost when the client disconnects through reloading the page, closing the tab or
+        navigating away from the page. It can be used to store data that is only relevant for the current view such
+        as filter settings on a dashboard or in-page navigation. As the data is not persisted it also allows the
+        storage of data structures such as database connections, pandas tables, numpy arrays, user specific ML models
+        or other living objects that are not serializable to JSON.
+        """
+        client = context.get_client()
+        return client.state
+
     def clear(self) -> None:
         """Clears all storage."""
         self._general.clear()
         self._users.clear()
+        if get_slot_stack():
+            self.client.clear()
```

Comment:
Is `client.clear()` correct? I can not find it.

Reply:
Yeap, it's correct. Property self.client returns Client.state which is an ObservableDict which declares .clear in observables.py:73

Reply:
Ah indeed!